docs/threat-model.md

```diff
@@ -630,7 +611,25 @@ Notable threats to the system that could impact assets.
 
 **Validation**: Integration testing with various client versions and protocol combinations
 
-**Cost**: Medium - Testing infrastructure and compatibility analysis
+**Cost**: MEDIUM - Testing infrastructure and compatibility analysis
+
+### M11: No Conflicting Transactions
+**Addressing threats**: T1, T12
+
+**Decision**: MITIGATE
+
+**Control type**: By design
+
+**Implementation**:
+- Protocol design inherently prevents conflicting transactions from reaching the chain
```

Comment:
Not 100% true any more, because of the need for execution bitmaps.

Reply:
True, my PR predates their introduction and I'm not fully on board with those yet :)